packages/jaeger-ui/src/components/TracePage/TracePageHeader/TracePageHeader.tsx

```diff
@@ -149,15 +149,17 @@ export function TracePageHeaderFn(props: TracePageHeaderEmbedProps & { forwarded
       return { ...rest, value: renderer(trace) };
     });
 
+  const traceShortID = trace.traceID.slice(0, 7);
+
   const title = (
     <h1 className={`TracePageHeader--title ${canCollapse ? 'is-collapsible' : ''}`}>
-      <TraceName traceName={getTraceName(trace.spans)} />{' '}
-      <small className="u-tx-muted">{trace.traceID.slice(0, 7)}</small>
+      <TraceName traceName={trace.traceName} /> <small className="u-tx-muted">{traceShortID}</small>
```

Comment:
please keep `{' '}`

Reply:
Sure, I think `yarn prettier` is what removed it though.

Reply:
I removed it, but `yarn prettier` put it back. Is it not only relevant when it's a trailing space at the end of line?

Reply:
ok do what prettier says otherwise the CI won't pass

Reply:
Done.